ssh/tailssh/tailssh.go

```diff
@@ -317,27 +286,23 @@ func (srv *server) handleAcceptedSSH(ctx context.Context, s ssh.Session, ci *ssh
 	return
 }
 
-func setWinsize(f *os.File, w, h int) {
-	syscall.Syscall(syscall.SYS_IOCTL, f.Fd(), uintptr(syscall.TIOCSWINSZ),
-		uintptr(unsafe.Pointer(&struct{ h, w, x, y uint16 }{uint16(h), uint16(w), 0, 0})))
-}
-
 type sshConnInfo struct {
-	// now is the time to consider the present moment for the
-	// purposes of rule evaluation.
-	now time.Time
+	StartTime time.Time
```

Comment:
missing docs on this one.

why are these all exported now?

Reply:
maybe do one separate PR where those are renamed, to get diff size down?

Reply:
I exported these when I was doing json marshaling to the child process, but that lead to really weird tty behavior. I can revert this bit, if you prefer